src/app/components/Video/index.jsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import styled from 'styled-components';
 import { videoComponentPropTypes } from '../../models/propTypes';
+import { GridItemConstrainedLargeNoMargin } from '../../lib/styledGrid';
+
+const MediaPlayerContainer = styled.div`
+  height: 26em;
+  width: 100%;
+`;
```

Comment:
The div that contains the mediaplayer needs to have its size defined or the media player will not show up! I chose 26em because it was almost the size used in the UX designs but this should obviously be something else. Would be good to discuss what we want to do with the height, the width should stay at 100% as it will then fill the `gridItem` that contains it.

Reply:
Has there been any update to this? @eleanombre @oluoluoxenfree

Reply:
Nope, I'll ask @nataliesmart now

Reply:
@nataliesmart suggested looking at the sizing when we get to #1733 @rhell, so will leave it as is for now